docs/source/simulation/simulation.rst

```diff
@@ -6,8 +6,7 @@ The following series of commands can be used to easily run a simulation with a t
 .. code-block:: console
 
         (venv)$ cd demo/Test
-        (venv)$ ./build_test_design.sh
-        (venv)$ ./run_simulation.sh
+        (venv)$ make
 
 FABulous comes with 3 different simulation methods _`configuration module`,
```

Comment:
We should mention all the required dependencies and installations for simulation here.